### PR TITLE
Structure membership information

### PR DESCRIPTION
**Pull request by cthoyt**
This pull request standardizes the information about membership into yaml files to make it more simple to maintain. It also adds the requirement that the ORCID identifier of each person is annotated, I went through and grabbed all of them.

This PR also makes minor changes to the layout of various WG pages for consistency (no information loss).

## To Do (but not necessary for merging)

Get ORCID identifiers for the following alumni (help wanted):

- [ ] Eric Douglass (@dougli1sqrd is this you?)
- [ ] Gareth Owen (this person? https://www.researchgate.net/scientific-contributions/Gareth-Owen-44854599 / https://mobile.twitter.com/GarethOwenLloyd?)
- [ ] Carlo Tornial (this person? https://scholar.google.com/citations?user=pYexO7AAAAAJ&hl=en?)

## Screenshots

Screen Shot 2022-10-01 at 13 46 43

Screen Shot 2022-10-01 at 14 24 18

### CODE REVIEW

File: _data/alumni.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+members:
+- name: Michael Ashburner
```
Comment (cthoyt) [inline _data/alumni.yml:2]: ideally we could also get departure dates for alumni, but I don't think that's feasible @nlharris

**Review by matentzn — COMMENTED** [review]
@cthoyt also always - this is amazing. Having the memberships in this much more formal way is a great idea: it gives us a bit more framework to communicate expectations to OBO Ops members. 

The only bigger thing that I disagree with is that the OBO Ops member list is now replaced by the other three sublists. I think there should be a single table with all OBO ops members, easily accessible, and, indeed, with photos even (I collected them all for ICBO, could upload them).

A smaller thing is the redundancy of having to curate the same affiliation and orcid twice if people are part of multiple subgroups. Also, we are about to add other groups, such as governance.

Do address all of the above at once, I recommend to have a single yaml file with all OBO Ops current members, and a list item: `working_groups` or some such which lists which working groups they are part of. The individual pages can then use this to decide if a person should be listed on page (liquid template conditional).

**Comment by cthoyt** [comment]
Done

Screen Shot 2022-10-01 at 16 33 17

File: _data/operations.yml [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+  affiliation: Lawrence Berkeley National Laboratory, Berkeley, CA, USA
+  orcid: "0000-0001-8244-1536"
+  groups:
+  - technical
+- name: Alexander Diehl
+  affiliation: University at Buffalo, Buffalo, NY, USA
+  orcid: 0000-0001-9990-8331
+  groups:
+  - technical
+
```
Comment (matentzn) [inline _data/operations.yml:68]:
```suggestion
```

**Review by matentzn — COMMENTED** [review]
This is absolutely fantastic. How exactly are these pages updated? Does this somehow happen automatically when you, say change, the opertions.yml file?

**Comment by cthoyt** [comment]
> This is absolutely fantastic. How exactly are these pages updated? Does this somehow happen automatically when you, say change, the opertions.yml file?

Yes, data inside the `_data` directory is used during building the static site. So anytime you commit a change to the `operations.yml` file, it will trigger a rebuild of the static site, and automatically reflect the changes there.

**Review by matentzn — APPROVED** [review]
I am very happy to take this for a ride, but your own QC is failing :D

**Comment by cthoyt** [comment]
gotta eat your own dog food sometimes

**Comment by cthoyt** [comment]
Unfortunately I got excited about using high-power typing and imported something that wasn't available on py36. But it's crazy to run tests on that, so I upgraded the testing pipeline to use py39 and it's all good now.

**Review by matentzn — APPROVED** [review]

**Pull request merged**